shell/components/ResourceDetail/Masthead.vue

```diff
@@ -435,7 +435,7 @@ export default {
         <div class="actions-container">
           <div class="actions">
             <button
-              v-if="detailsAction"
+              v-if="detailsAction && currentView === 'detail' && isView"
```

Comment:
`'detail'` --> the const `_DETAIL` (need to pass it through to template via `data`)

Reply:
Done